coffee-chats/src/main/webapp/src/components/ConnectBackCard.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import {Card, CardContent, CardActions, Button, Typography} from "@material-ui/core";
+import {humanReadableJoin} from "../format";
+
+function ConnectBackCard(props: { names: string[], tags: string[] }) {
+  return (
+      <Card>
+        <CardContent>
+          <Typography variant="h5" gutterBottom>
+            Connect Back
+          </Typography>
+          <Typography color="textSecondary" gutterBottom>
+            You've recently chatted
+            about {humanReadableJoin(props.tags)} {}
+            with {}
+            <Typography color="textPrimary" component={'span'} >
+              {humanReadableJoin(props.names)}
+            </Typography>.
+          </Typography>
+        </CardContent>
+        <CardActions>
+          <Button size="small">More</Button>
+        </CardActions>
+      </Card>
+  )
+}
+
+export default ConnectBackCard;
```

Comment:
Google Styleguide says to not use default exports unfortunately, despite React's tendency to use them. Perhaps  @DarthGandalf can clarify whether to follow the Google JS styleguide, or stick with the React convention?

Reply:
It should be easy enough to follow Google guide by adding {} around the name here and on import side